### PR TITLE
Created Users Model Table

### PR DESCRIPTION
**Pull request by tanvinerkar**

### CODE REVIEW

File: lifeiitkbackend/settings.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
-    'django.contrib.staticfiles',
```
Comment (ditttu) [inline lifeiitkbackend/settings.py]: Why did you remove this?

**Comment by ditttu** [comment]
Also...resolve conflicts in gitignore

**Comment by bhuvansingla** [comment]
Resolved :)

**Review by ditttu — APPROVED** [review]
